connect/src/spirc.rs

```diff
@@ -1289,6 +1307,65 @@ impl SpircTask {
         Ok(())
     }
 
+    async fn load_context_from_uri(
+        &mut self,
+        context: &Option<Context>,
+        context_uri: &str,
```

Comment:
You could consider using `impl Into<String>` here.

Reply:
I look a bit into that, but it seems that I only require a reference in the method, so it seems preferred to keep it a `&str`.

Reply:
Oh, good point. I removed the previous error. Will check if that was intentional, as it's a while back that I made the change^^;

Reply:
Ah, I think now I see what you meant when you suggested the change back then. Making it a `String` makes more sense as the original input is dropped afterwards anyways, so might as well return the `uri` as is without going through the `String` to `&str` to `String` conversion. Good find :D